repo/packages/C/calico/2/config.json

```diff
@@ -0,0 +1,245 @@
+{
+  "properties": {
+    "Framework Settings": {
+      "description": "Configuration options for Calico's DC/OS Framework.",
+      "properties": {
+        "framework-name": {
+          "type": "string",
+          "description": "The name of the framework.",
+          "default": "calico-install-framework"
+        },
+        "master": {
+          "default": "zk://master.mesos:2181/mesos",
```

Comment:
I switched default to `zk://zk-1.zk:2181/calico`, allowing the user to expand it to the other zk's if they're running a multi cluster. Is that sufficient, or should it be the full string?